Services/CO.CDP.Login.WebApi.Tests/FakeOneLoginControllerTests.cs

```diff
@@ -0,0 +1,38 @@
+using System.Net;
+using System.Net.Http;
+using Xunit;
+using System.Text.Json;
+using CO.CDP.Common.Auth;
+
+namespace CO.CDP.Login.WebApi.Tests;
+
+public class FakeOneLoginControllerTests
+{
+    private readonly HttpClient _client;
+
+    public FakeOneLoginControllerTests()
+    {
+        _client = new HttpClient();
+    }
+
+    [Fact]
+    public async Task Get_HealthyEndpoint_ReturnsSuccess()
+    {
+        string url = "http://localhost:5130/fake-onelogin";
+        HttpResponseMessage response = await _client.GetAsync(url);
+        string contentString = await response.Content.ReadAsStringAsync();
+        var result = JsonSerializer.Deserialize<OneLoginResponce>(contentString, new JsonSerializerOptions { PropertyNameCaseInsensitive = true });
```

Comment:
Line 23,24 can be merged as 
await response.Content.ReadFromJsonAsync<OneLoginResponce>()